src/pages/studyView/queryStudyAnalytics.ts

```diff
@@ -0,0 +1,24 @@
+import * as request from "superagent";
+
+export interface IAnalyticsResponse {
+    totalUses: number;
+    totalUniqueUses: number;
+    timesVisited: number;
+    timesUniqueVisited: number;
+}
+
+export interface IQueryResponse extends request.Response {
+    body: IAnalyticsResponse;
+}
+
+export default async function queryStudyAnalytics(
+    studyIds: string[],
+    apiURL: string
+): Promise<IAnalyticsResponse> {
+    try {
+        const res: IQueryResponse = await request.get(`${apiURL}?studyName=${studyIds[0]}`);
```

Comment:
I see, we're making up for fact that we don't have SuperAgent typings.   I'm sure they exist.  Anyways, this is totally not important.